lib/foreman_maintain/repository_manager/el.rb

```diff
@@ -0,0 +1,43 @@
+module ForemanMaintain::RepositoryManager
+  class El
+    include ForemanMaintain::Concerns::OsFacts
+    include ForemanMaintain::Concerns::SystemHelpers
+
+    def disable_repos(repo_ids)
+      if el7?
+        execute!("yum-config-manager --disable #{repo_ids.join(',')}")
+      else
+        execute!("dnf config-manager --set-disabled #{repo_ids.join(',')}")
+      end
+    end
+
+    def enable_repos(repo_ids)
+      if el7?
+        execute!("yum-config-manager --enable #{repo_ids.join(',')}")
+      else
+        execute!("dnf config-manager --set-enabled #{repo_ids.join(',')}")
+      end
+    end
+
+    def enabled_repos_hash
+      yum_cmd = 'yum repolist enabled -d 6 -e 0 2> /dev/null'
+      repos = execute(yum_cmd)
+      return {} if repos.empty?
+
+      repo_hash = Hash[*repos.delete!(' ').split("\n").grep(/Repo-id|Repo-baseurl/)]
```

Comment:
This use to include `Repo-id` and `Repo-baseurl` and then later on when required we called `trim_repo_ids`. As these labels are not required I thought to remove those.